app/assets/stylesheets/expenses.css.scss

```diff
@@ -0,0 +1,7 @@
+#create_expense_link {
```

Comment:
Avoid using id selectors<br>Selector `create_expense_link` should be written in lowercase with hyphens